src/cli/commands/force/lightning/local/device/create.ts

```diff
@@ -165,6 +163,20 @@ export class Create extends Setup {
             });
     }
 
+    private _requirements: CommandRequirements = {};
+    public get commandRequirements(): CommandRequirements {
+        if (Object.keys(this._requirements).length === 0) {
+            const commandDict: CommandRequirements = {};
```

Comment:
nit: rename `commandDict` to `requirements` since that's more representative of what it is (in here and other files)